### ISSUE TITLE
Failure to parse output from SSMS

### ISSUE THREAD

**Issue by n2ygk** [xnuinside/simple-ddl-parser#91]
**Describe the bug**
Here's an example of (silently) failing to parse DDL generated by MS SQL Server Management Studio. Other DDL output from SSMS parses just fine. So I'm not sure where this one is failing.

**To Reproduce**
```
sdp aud.tcal_tgt.Table.sql
```

**Expected behavior**
Out in `schemas/aud_schema.json` that contains the table info.

**Additional context**
Try it with this:
```
USE [sasgolddevdb]
GO
/****** Object:  Table [aud].[tcal_tgt]    Script Date: 11/11/2021 11:18:41 AM ******/
SET ANSI_NULLS ON
GO
SET QUOTED_IDENTIFIER ON
GO
CREATE TABLE [aud].[tcal_tgt](
	[TCAL_SID] [decimal](30, 0) NOT NULL,
	[TERM_YR] [varchar](4) NULL,
	[TERM_CD] [varchar](4) NULL,
	[CAL_CD] [varchar](2) NULL,
	[LMOD_USER] [varchar](4) NULL,
	[LMOD_DT] [datetime2](7) NULL,
	[TERM_BEGIN_DT] [datetime2](7) NULL,
	[TERM_END_DT] [datetime2](7) NULL,
	[EFF_EXP_DT] [datetime2](7) NULL,
	[EFF_EXP_FLG] [varchar](1) NULL,
	[FEED_OPR] [varchar](1) NULL,
	[REC_CREATED_DT] [datetime2](7) NULL,
	[REC_CREATED_BY] [varchar](25) NULL,
	[REC_MODIFIED_DT] [datetime2](7) NULL,
	[REC_MODIFIED_BY] [varchar](25) NULL,
	[REC_DELETED_DT] [datetime2](7) NULL,
	[REC_DELETED_BY] [varchar](25) NULL,
	[SYSTEM_OPERATION] [varchar](1) NULL,
PRIMARY KEY CLUSTERED 
(
	[TCAL_SID] ASC
)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON, FILLFACTOR = 80) ON [PRIMARY]
) ON [PRIMARY]

GO
```

**Comment by xnuinside** [comment]
@n2ygk I checked and I see, that just was not added support for PRIMARY KEY CLUSTERED  without constraint name before it. Working on it right now, I'm planning that I will release a new version tomorrow evening with fixes for both issues

**Comment by xnuinside** [comment]
@n2ygk fixes for both issues were released in version 0.22.0 - https://pypistats.org/packages/simple-ddl-parser . If you will find anything more - please feel free to open new issue and thank you one more time!
